modules/cudf/src/data_frame.ts

```diff
@@ -203,11 +204,11 @@ export class DataFrame<T extends TypeMap = any> {
    *
    * @example
    * ```typescript
-   * import {DataFrame} from '@rapidsai/cudf';
+   * import {DataFrame, Series} from '@rapidsai/cudf';
    *
    * const df = new DataFrame({a: [1, 2, 3]});
    *
-   * df.assign({b: ["foo", "bar", "bar"]})
+   * df.assign({b: Series.new(["foo", "bar", "bar"])})
```

Comment:
Assign uses a `SeriesMap`, which doesn't allow for a syntax such as...
```
df.assign({b: ["foo", "bar", "bar"]})
```
Maybe we should consider adding this.

Reply:
Good catch, this docstring was wrong.